iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -55,11 +55,13 @@ UnixDomainSocket::UnixDomainSocket(UnixDomainSocket&& other) noexcept
     *this = std::move(other);
 }
 
-// NOLINTNEXTLINE(readability-function-size) todo(iox-#832): make a struct out of arguments
+// @todo(iox-#832): make a struct out of arguments
```

Comment:
I think this kind of struct would be detrimental fr usability.

Reply:
I would suggest to be pragmatic here. Certainly, introducing arbitrary `struct`s doesn't make sense. Combining parameters that belong together, however, does make sense. Here, maybe name + properties, although name could be considered a property, too.

Reply:
The last two parameters will still be easily swappable. Here, we already agreed that all those places in the code will be addressed together and use `NewType`, right?

Reply:
IMO this considerations should be part of the refactoring, therefore I added to #832.